app/models/miq_cockpit_ws_worker.rb

```diff
@@ -0,0 +1,73 @@
+require 'miq_apache'
+require 'runcmd'
+require 'linux_admin'
```

Comment:
Where are linux_admin or runcmd used in this file?  I tried searching but I don't see it.